featurebyte/feast/enum.py

```diff
@@ -29,7 +29,7 @@ def to_feast_primitive_type(dtype: DBVarType) -> PrimitiveFeastType:
     mapping = {
         DBVarType.BOOL: PrimitiveFeastType.BOOL,
         DBVarType.VARCHAR: PrimitiveFeastType.STRING,
-        DBVarType.FLOAT: PrimitiveFeastType.FLOAT64,
+        DBVarType.FLOAT: PrimitiveFeastType.FLOAT32,
```

Comment:
If we make it `FLOAT64`, we should expect the value is `np.float64` during feature materialization. What happens to unity catalog is that the `np.float32` is returned for certain features. It will hit assertion error during Feast's type inference: https://github.com/feast-dev/feast/blob/master/sdk/python/feast/type_map.py#L430-L432